### PR TITLE
VBF blikelihood clean

### PR DESCRIPTION
**Pull request by chernyavskaya**
Only files that are relevant to VBF blikelihood added. In PhysicsTools/Heppy/python/physicsobjects/Jet.py  : self.qgl_value=-1. #if no qgl calculator configured  (zero return value was changed to -1 because 0 - is the value that QGL has if it is calculated and is gluon thus cannot be used here )

### CODE REVIEW

**Comment by arizzi** [comment]
the PR is ok, the only thing that is not exactly nice is that it is made of a zillion not always very descriptive commits. Can you try to squash them all in a single (or anyhow few) commits? (if not, I can do that)

PS: I know this seems just annoying but it would greatly help the maintainance and would allow us all to understand what's going on when looking back in a few months.

**Comment by chernyavskaya** [comment]
Andrea, could you please help me with squashing ? The problem is that there is a merge in the middle and I cannot figure out how to squash properly...I am sorry.

**Comment by arizzi** [comment]
probably the easiest thing is 
-take a clean area
-copy the files you modified from the working area
- commit them

if you do so please be sure you commit the changes outside VHbbHeppy (e.g. Jet.py) in a separate commit starting with "Heppy: ....description here..."

File: PhysicsTools/Heppy/python/physicsobjects/Jet.py [hunk 1]
```diff
@@ -133,7 +133,7 @@ def qgl(self) :
               self.computeQGvars()
               self.qgl_value=self.qgl_calc(self,self.qgl_rho)
 	  else :
-              self.qgl_value=0. #if no qgl calculator configured
+              self.qgl_value=-1. #if no qgl calculator configured
```
Comment (arizzi) [inline PhysicsTools/Heppy/python/physicsobjects/Jet.py]: I make a separate commit for this myself
